cmd/kwild/server/build.go

```diff
@@ -359,6 +363,20 @@ func (c *closeFuncs) closeAll() error {
 	return err
 }
 
+func verifyDependencies(d *coreDependencies) {
+	// Check if pg_dump is installed, which is necessary for snapshotting during migrations and when snapshots are enabled
+	if _, err := exec.LookPath("pg_dump"); err != nil {
+		failBuild(err, "pg_dump is not installed. Please install it to use snapshots or migrations.")
+	}
+
+	if d.cfg.ChainConfig.StateSync.Enable {
+		// Check if psql is installed, which is required for state-sync
+		if _, err := exec.LookPath("psql"); err != nil {
```

Comment:
I'm curious what the output of the following is on the mac versions:

```
$  psql -V
psql (PostgreSQL) 16.4 (Ubuntu 16.4-1.pgdg22.04+1)

$  pg_dump -V
pg_dump (PostgreSQL) 16.4 (Ubuntu 16.4-1.pgdg22.04+1)
```

Reply:
```
$ psql -V
psql (PostgreSQL) 16.4
```

Reply:
Added version checks to ensure that everyone is running 16.x.